docs/feature_unicode.md

```diff
@@ -1,56 +1,78 @@
 # Unicode Support
 
-There are three Unicode keymap definition methods available in QMK:
+Unicode characters can be input straight from your keyboard! There are some limitations, however.
 
-## `UNICODE_ENABLE`
+There are three different methods available in QMK:
 
-Supports Unicode up to `0x7FFF`. This covers characters for most modern languages, as well as symbols, but it doesn't cover emoji. The keycode function is `UC(c)` in the keymap, where _c_ is the code point's number (preferably hexadecimal, up to 4 digits long). For example: `UC(0x45B)`, `UC(0x30C4)`.
+## Basic Unicode
 
-## `UNICODEMAP_ENABLE`
+This method supports Unicode codepoints up to `0x7FFF`. This covers characters for most modern languages, as well as symbols, but it doesn't cover emoji.
```

Comment:
```suggestion
This method supports Unicode code points up to `0x7FFF`. This covers characters for most modern languages, as well as symbols, but it doesn't cover emoji.
```
“Code point” is usually two words. It's the preferred spelling in the Unicode standard.

Reply:
Could have sworn it was the other way around...